src/backend/db/categories.js

```diff
@@ -5,8 +5,18 @@
 
 export const categories = [
   {
-    _id: "",
-    categoryName: "",
-    description: "",
+    _id: "stvcat001",
+    categoryName: "Bird Quiz",
+    icon: "fa-dove",
+  },
+  {
```

Comment:
Suggestion : keep category without spaces , may cause issue with the category urls , and in the urls it will come like %20  , can use another attribute

Reply:
I will be using the id for fetching